rasa/train.py

```diff
@@ -163,31 +184,43 @@ def train_core(
 
     _train_path = train_path or tempfile.mkdtemp()
 
+    story_directory = data.get_core_directory(stories)
```

Comment:
if a train path was passed we don't have to validate this since the `train` function for the stack model already did these validations.